packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -1799,10 +1804,11 @@ class _SubmenuButtonState extends State<SubmenuButton> {
         // TextButton, we merge the styles, merging them in the right order when
         // each type of style exists. Each "*StyleOf" function is only called
         // once.
-        final ButtonStyle mergedStyle =
-            widget.style?.merge(widget.themeStyleOf(context)?.merge(widget.defaultStyleOf(context))) ??
-                widget.themeStyleOf(context)?.merge(widget.defaultStyleOf(context)) ??
-                widget.defaultStyleOf(context);
+        ButtonStyle mergedStyle = widget.themeStyleOf(context)?.merge(widget.defaultStyleOf(context))
```

Comment:
Same here...

Reply:
Similarly here, just to handle the case when we have the `widget.style` but don't have `themeStyleOf`:)